### ISSUE TITLE
Custom snippet tab stop doesn't work

### ISSUE THREAD

**Issue by Matticusau** [microsoft/vscode-docs-authoring#140]
I've been finding that using the **docs authoring pack** and in particular the **docs-markdown** ext prevents the Tab Stop functionality in other custom snippets that I have.

Steps to replicate

I created a custom snippet following [https://code.visualstudio.com/docs/editor/userdefinedsnippets](https://code.visualstudio.com/docs/editor/userdefinedsnippets) such as:

```
{
	"My Custom Snippet": {
		"prefix": "mysnippet",
		"scope": "markdown",
		"body": [
			"First tab stop: $1",
			"Second tab stop: $2",
			"Last tab stop: $0"
		],
		"description": "Add a row to the Learning Plan Table"
	}
}
```

Create a new file in vscode, set the language mode to markdown. 
Type `mysnippet` and press `CTRL+SPACE` then select the snippet and press `ENTER`
Type text in the first tab stop, then press tab.

It will insert a tab and not move onto the next tab stop as per this image.

![image](https://user-images.githubusercontent.com/11083642/52788361-66e06180-30ac-11e9-8489-8e707dca1f42.png)

If I disable the **docs-markdown** this allows the tabstop to work. I can also disable the whole **docs authoring pack** and it resolves the issue. 

For the moment I have disabled this for the workspace but I could see the need to use this extension within my current project along with custom snippets.

VS Code version info
Version: 1.31.1 (user setup)
Commit: 1b8e8302e405050205e69b59abb3559592bb9e60
Date: 2019-02-12T02:20:54.427Z
Electron: 3.1.2
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763

Extension versions:
docs-authoring-pack: 0.1.1
docs-markdown: 0.2.2

**Comment by jamarw** [comment]
Hi @Matticusau , I'll start looking into this today.  Thanks

**Comment by jamarw** [comment]
Hi @Matticusau , this issue was fixed in release 0.2.73. Thanks!
